client/components/account-details/banner.tsx

```diff
@@ -0,0 +1,65 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import InlineNotice from 'wcpay/components/inline-notice';
+import {
+	AccountDetailsData,
+	BannerBackgroundColor,
+} from 'wcpay/types/account/account-details';
+import { getIconByName } from './utils';
+
+interface BannerProps {
+	banner: AccountDetailsData[ 'banner' ];
+}
+
+const getNoticeStatusFromColor = ( color: BannerBackgroundColor ) => {
+	switch ( color ) {
+		case 'green':
+			return 'success';
+		case 'blue':
+			return 'info';
+		case 'yellow':
+			return 'warning';
+		case 'red':
+			return 'error';
+		default:
+			return 'info';
```

Comment:
Maybe later I'll be able to answer this question myself, but why are we doing this?

I would understand if we wanted to be able to provide custom colors from the server, yet still use standardized ones for most cases. However, those colors are fixed. Given that the feature is not online yet, can't we just provide the right colors from the server, e.g. `success` rather than `green`?